c31-pryhaven/nutshell2
# My event edit

# Description

Created an edit button and form for the user to be able to change details about the event and then put the changes to the API.

Applies to issue # (issue number)
3.4
## Type of change

Please delete options that are not relevant.


# Testing Instructions

run git fetch --all in terminal
pull down my-event-edit from github.
take data from the example database and plug it into nutshell.json.
run the command json-server -p 8088 -w nutshell.json at ./nutshell2/api
go to root path and run npm start
go to browser and click on events on the nav bar.
click on the edit button and edit anything about the event.
hit the submit button and you should be redirected to the events page with your newly added event.




# Checklist:
